lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -86,37 +92,55 @@ def configure(conf)
         end
       end
 
-      unless @use_metadata_service
-        unless @project_id && @zone && @vm_id
-          raise Fluent::ConfigError,
-              ('Please specify "project_id", "zone" and "vm_id" if you set "use_metadata_service" to false')
+      if @use_metadata_service
+        if @project_id.nil?
+          begin
+            @project_id = fetch_metadata('project/project-id')
+          rescue
```

Comment:
It will be reported, but my question was if the user will be able to easily distinguish between 
"oops I forgot to set project_id" and "fetch_metadata returned HTTP error".